src/java/Controller/SendEmail.java

```diff
@@ -182,60 +243,168 @@ else if(count ==1)
 		return false;
 	}
     
-    static String getEmpId(String username) {
+    static String[] getEmployee(String email) {//getEmployee returns an array of attributes
     	
     	try
-		{
-			Class.forName("oracle.jdbc.driver.OracleDriver"); // Type 4 Driver Pure Java Driver
-			Connection con = DriverManager.getConnection("jdbc:oracle:thin:@localhost:1521:XE","Student_Performance","Student_Performance");
-			Statement st = con.createStatement();
-			ResultSet rs = st.executeQuery("select employee_id from employees where email = " + "'" + username + "'");
+	{
+            Class.forName("oracle.jdbc.driver.OracleDriver"); // Type 4 Driver Pure Java Driver
+            Connection con = DriverManager.getConnection("jdbc:oracle:thin:@localhost:1521:XE","Student_Performance","Student_Performance");

```

Comment:
This shit should be in a config file somewhere, with the `class.forname` and setup and stuff run as part of a setup. but absolutely passwords need to be in config so they can be encrypted later... not by up i hope